packages/client/tests/e2e/_utils/run.ts

```diff
@@ -33,6 +33,7 @@ async function main() {
   args['--verbose'] = args['--verbose'] ?? false
   args['--skipBuild'] = args['--skipBuild'] ?? false
   args['--clean'] = args['--clean'] ?? false
+  const filters = args._.flatMap((filter) => [`../*${filter}*/_steps.ts`, `../*${filter}*/**/_steps.ts`])
```

Comment:
allows to run singular e2e test.

Reply:
Is this equivalent to this line https://github.com/prisma/prisma/pull/19616/files#diff-9f3d1a92028dfbb4e992fc36c89be444605b3a820dc36ef26207e31f6febeab0R110?

Reply:
Oh, right, missed that part, will restore to original then